stripe/src/main/java/com/stripe/android/paymentsheet/BasePaymentMethodsListFragment.kt

```diff
@@ -36,9 +35,6 @@ internal abstract class BasePaymentMethodsListFragment(
 
         this.config = nullableConfig
 
-        // reset the mode in case we're returning from the back stack
-        sheetViewModel.updateMode(SheetMode.Wrapped)
```

Comment:
This is still needed to update the close/back button.

1. Custom sheet w/customer
2. Add a card
3. Press the back button
Expected: The list should be shown with the close button instead of the back button

Reply:
This is an existing issue, right? There are a few other issues with the custom sheet (like the keyboard covering the content), I'll address all of them on the next PR.

Reply:
This is new to this PR because you need that line 40.   When you update the Mode in BaseSheetActivity it checks the backstack and sets the button.

Reply:
You're right, this change should actually be part of the next PR. Done.